VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -35,21 +42,33 @@ protected void onCreate(Bundle savedInstanceState) {
 
         webView = (WebView) findViewById(R.id.webview);
         webView.loadUrl(environment.getConfig().getCourseDiscoveryConfig().getCourseSearchUrl());
+
+        DrawerLayout mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+        mDrawerToggle = new ActionBarDrawerToggle(this, mDrawerLayout,
```

Comment:
Hmm, since we're not actually using the toggle functionality here, I guess you could just have used `DrawerLayout.SimpleDrawerListener`.